Tests/SentryProfilerTests/SentryProfilerSwiftTests.swift

```diff
@@ -175,6 +169,8 @@ class SentryProfilerSwiftTests: XCTestCase {
             renderGPUFrame(.normal)
             renderGPUFrame(.frozen)
             renderGPUFrame(.normal)
+            changeFrameRate(.high)
+            renderGPUFrame(.normal)
```

Comment:
Always start and end on opposite frame rates, so the frame rate change gets picked up by the frames tracker between its previous and current callback invocations.